controllers/utils/routeStatic.js

```diff
@@ -34,20 +35,54 @@ let servePage = (page, router) => {
 
 let serveCmsPage = (page, sectionId, router) => {
     router.get(page.path, (req, res) => {
-        const renderPage = content => {
-            res.render('pages/legacy', {
-                title: content.title,
-                content: content
-            });
-        };
+        /**
+         * Placeholder images
+         * @TODO: Replace with different ones, or ideally fetch from CMS.
```

Comment:
Not super keen on this living here – can we put this in its own config file as part of this PR?